lib/Template/nodes/TemplateNodeComponent.js

```diff
@@ -101,6 +212,29 @@ export class TemplateNodeComponent extends TemplateNode {
     }
 
     /**
+     * @returns {{}}
+     */
+    getPragma() {
+        let pragma = {};
+
+        this.attributes.forEach(attr => {
+            if (attr.name.slice(0, 1) === '#') {
+                let name = attr.name.slice(1);
+
+                if (attr.value instanceof TemplateNodePrimitive) {
```

Comment:
the `attr.value` used 4 times. you can extract it to separate variable. `let value = attr.value;` to reduce code length :)